cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -88,6 +88,12 @@ check_include_files("linux/memfd.h" HAVE_LINUX_MEMFD)
 if(HAVE_LINUX_MEMFD)
   list(APPEND ARCH_DEFINES "-DHAVE_LINUX_MEMFD=1")
 endif()
+
+check_include_files("linux/udmabuf.h" HAVE_LINUX_UDMABUF)
```

Comment:
Can you add a warning if this is missing so we can verify in compile logs after the fact?

Reply:
added